### PR TITLE
use quote instead of string formatting in process_create_migration

### LINKED ISSUE #2: Use `quote` to generate Rust code instead of string format
Hi there,
thank you for creating this project, it looks super useful!

While going through the code generation done in `g`, I noticed that the Rust code generation is done through plain string formatting. I also worked on a small project where I needed to generate Rust code and I resorted to this approach at first as well. However, I quickly ran into problems like code generation being dependent on conditionals quickly resulting in annoying string concatenations.

I decided to use the popular [quote](https://github.com/dtolnay/quote#syntax) library to template Rust code in a clean manner, making my life a lot easier and the code cleaner to read (I also used [prettyplease](https://github.com/dtolnay/prettyplease) to auto-format the generated code).

Maybe this is helpful, but feel free to close if this project goes with the format approach.

My project is here for reference: https://github.com/JonasCir/java2rs

### PR DESCRIPTION
**Pull request by JonasCir**
fixes #2

### CODE REVIEW

File: src/g/process.rs [hunk 1]
```diff
@@ -29,72 +30,74 @@ pub async fn process_create_migration(model: &str) {
...
     let file_path = String::from(file_dir) + &filename + ".rs";
-    let file_content = format!(
-        "use entity::{};
-use sea_orm::{{DbBackend, EntityTrait, Schema}};
-use sea_orm_migration::prelude::*;
 
-pub struct Migration;
+    // create tokens used in the quasi-templating below
+    let model_name_ident = format_ident!("{}", model);
+    let file_name_lit: syn::LitStr = syn::parse_str(&filename).expect("Unable to parse println!() argument");
```
Comment (POPPIN-FUMI) [inline src/g/process.rs:36]: line 36 gets this error below;

`thread 'main' panicked at 'Unable to parse println!() argument: Error("expected string literal")'`

Can you try `cargo run g model test` command to test if this line works?
Thank you!

Comment (JonasCir) [reply src/g/process.rs:36]: sure I think I will find time over the weekend

**Comment by JonasCir** [comment]
@POPPIN-FUMI You closed this without merging was this intentional? :D

Sure I can go over other methods when I have time.

**Comment by JonasCir** [comment]
Not certain what your workflow is here. If you just cherry pick my commits without merging the PR I'm fine as well, right now I'm a little bit confused :smiley:

**Comment by JonasCir** [comment]
Great and all good :)
thank you for following up on this!

**Pull request closed without merging**